### PR TITLE
Feature/run pylint

### PR DESCRIPTION
**Pull request by pcrespov**
- fixed pylint setup. Created new configuration file .pylintrc
- reviewed and fixed all pylint failures in python code

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
     "files.trimTrailingWhitespace": true,
-    "editor.tabSize": 2,
+    "editor.tabSize": 4,
```
Comment (odeimaiz) [inline .vscode/settings.json]: :-1:

Comment (pcrespov) [reply .vscode/settings.json]: actually it is based on pylintrc ... but I am flexible. You want 2?

Comment (odeimaiz) [reply .vscode/settings.json]: ¡A mí me daban dos!

Comment (oetiker) [reply .vscode/settings.json]: I personally prefer 4 too ... as this makes for better visibility of blocks and it discurrages deep nesting :)

**Pull request merged**